src/content/docs/design/design-with-code/design-with-style-hooks.mdx

```diff
@@ -0,0 +1,335 @@
+---
+page_id: 94a22da4-2013-4253-99dc-d686861cc320
+title: Style hooks 
```

Comment:
@clairekinde11 maybe `CSS style hooks`? @chris-pearce would know better but I feel like it needs some context